src/dashboard/Data/Browser/Browser.react.js

```diff
@@ -402,7 +402,7 @@ class Browser extends DashboardView {
           if (name === 'objectId' || this.state.isUnique && name !== this.state.uniqueField) {
             return;
           }
-          if (!!required) {
```

Comment:
I am always critical of this rule, because it's not a mere style change but a code change. I would suggest to disable this rule?

Reply:
I guess it's fine.